targets/TARGET_TOSHIBA/TARGET_TMPM4G9/reset_reason_api.c

```diff
@@ -0,0 +1,123 @@
+/* mbed Microcontroller Library
+ * (C)Copyright TOSHIBA ELECTRONIC DEVICES & STORAGE CORPORATION 2018 All rights reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stdbool.h>
+#include "reset_reason_api.h"
+#include "TMPM4G9.h"
+
+static uint8_t set_bit_count(uint32_t reg);
+static uint8_t bit_pos(uint32_t reg);
+static bool bit_status(uint32_t reg, uint8_t bit_no);
+
+static reset_reason_t reset_reason1[6] = {
+    RESET_REASON_POWER_ON,
+    RESET_REASON_UNKNOWN,
+    RESET_REASON_UNKNOWN,
```

Comment:
duplicated (line 26 and 27)

Reply:
@0xc0170 This array is used to provide value for corresponding bit position. If duplicate value is taken out from this array, then logic for getting the corresponding bit position value will fail.

Reply:
OK understood